protocol.md

```diff
@@ -345,6 +345,54 @@ between different platforms, but the process of communicating CENs between
 platforms has been reduced to working implementations in the above
 repositories.
 
+## CEN Reporting
+
+Given a specific choice of H, the above reduces to a CEN Report with less than
+1K per report, using JSON Object serialization/deserialization:
+
+```
+    report: ByteArray       // TBD based on Symptoms / Infections bits, considered in second
+    reportMetadata: String  // TBD, to support different kinds of reports, could be MIME type
+    L: String,     // short label L from CenKey. [hexadecimal form, 64 chars]
+    key: String,   // Key_{j-1}.                 [hexadecimal form, 64 chars]
+    j : Int,       // initial period j
+    jMax : Int     // number of periods
+```
```

Comment:
Because the reports are downloaded by all clients, we should ensure they are as compact as possible.  I think this format is not particularly space-efficient.  For instance, `L` and `key` should be encoded as bytes, not hex-encoded strings, and we may want to use LEB128 to encode `j` and `jMax`.

Reply:
I think it might be worth making a separate issue to figure out exactly what data we need to include in a report.

Reply:
#17